README.md

```diff
@@ -2319,6 +2319,41 @@ ANSWERS.addComponent('MyCustomComponent', {
 });
 ```
 
+# Custom Partials
+
+[Handlebars partials](https://handlebarsjs.com/guide/partials.html) are
+templates that can be reused and called directly from other templates. The 
```

Comment:
We need to be careful with the verbiage here. Templates have a very specific meaning in Handlebars and are distinct from Partials. A partial can be called from a template or another partial.

Reply:
Good to know! I've reworded this a little in the latest commit, but let me know if I should change it more